src/sinks/mezmo.rs

```diff
@@ -26,8 +26,41 @@ const PATH: &str = "/logs/ingest";
 
 /// Configuration for the `logdna` sink.
 #[configurable_component(sink("logdna"))]
+#[configurable(metadata(
+    deprecated = "The `logdna` sink has been renamed. Please use `mezmo` instead."
+))]
 #[derive(Clone, Debug)]
-pub struct LogdnaConfig {
+pub struct LogdnaConfig(MezmoConfig);
+
+impl GenerateConfig for LogdnaConfig {
+    fn generate_config() -> toml::Value {
+        <MezmoConfig as GenerateConfig>::generate_config()
+    }
+}
+
+#[async_trait::async_trait]
+impl SinkConfig for LogdnaConfig {
+    async fn build(
+        &self,
+        cx: SinkContext,
+    ) -> crate::Result<(super::VectorSink, super::Healthcheck)> {
+        warn!("The `logdna` sink has been renamed. Please use `mezmo` instead.");
```

Comment:
```suggestion
        warn!("DEPRECATED: The `logdna` sink has been renamed. Please use `mezmo` instead.");
```

Per the deprecation policy. Including this keyword makes it easy to grep for.